File: Python PowerPoint Examples/MasterSlidesApiDemo/getMasterSlide.py
import spirecloudpowerpoint
from spirecloudpowerpoint.configuration import *

appId = "your id"
appKey = "your key"
baseUrl = "https://api.e-iceblue.cn"
configuration = Configuration(appId, appKey, baseUrl)
api = spirecloudpowerpoint.api.master_slides_api.MasterSlidesApi(configuration)

name = "GetMasterSlide.pptx"
password = ""
storage = ""
folder = "input"
slideIndex = 0

result = api.get_master_slide(name=name, slide_index=slideIndex,
                              password=password, storage=storage,
                              folder=folder)

File: PHP PowerPoint Examples/MasterSlidesApiDemo/getMasterSlide.php
<?php

use Spire\Cloud\Powerpoint\Sdk\Api\MasterSlidesApi;
use Spire\Cloud\Powerpoint\Sdk\Configuration;

$appId = "your id";
$appKey = "your key";
$baseUrl="https://api.e-iceblue.cn";
$configuration = new Configuration($appId, $appKey,$baseUrl);
$apiInstance = new MasterSlidesApi($configuration);

$name = "GetMasterSlide.pptx";
$folder = "input";
$password = null;
$storage = null;
$slideIndex = 0;
$result = $apiInstance->getMasterSlide($name,$slideIndex, $password, $folder, $storage);
print_r($result);